test/plugin/scenarios/redisson-scenario/bin/startup.sh

```diff
@@ -18,4 +18,4 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -Dredis.servers=${REDIS_SERVERS} -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
\ No newline at end of file
+java -Dredis.servers=${REDIS_SERVERS} -Dskywalking.plugin.redisson.trace_redis_parameters=true -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
```

Comment:
As you activate this, please update the expected segment to verify this new tag.

Reply:
The same as above.